### PR TITLE
Placeholder for release 3.6.0

### PR DESCRIPTION
**Pull request by BFH-ktt1**

### CODE REVIEW

**Comment by MarcinOrlowski** [comment]
You may want to make it draft

**Comment by BFH-ktt1** [comment]
@MarcinOrlowski : I created a branch rc_3_6_0 with the current head. The development branch I gave the version number 3.7.0-develop. If you agree unblock this PR, and than we can continue as normal on the develop branch.

File: gradle.properties
Comment (MarcinOrlowski) [inline gradle.properties]: Not sure `develop` suffix is a best choice. I'd rather use `rc1` here, and the rationale is that `develop` means nothing really while `rc` is commonly used. Also `develop` is the branch name and nightly builder adds branch name to artifacts file names so double `develop` may look odd.

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

File: gradle.properties
Comment (BFH-ktt1) [reply gradle.properties]: So I just make 3.7.0 without extention.

Comment (MarcinOrlowski) [reply gradle.properties]: I think that's fine. As long as we release from `rc_3_6_0` (or from `master` with merged `rc_3_6_0`) we should be good. Version in `develop` is fine.

**Review by MarcinOrlowski — APPROVED** [review]

**Comment by MarcinOrlowski** [comment]
Hold on! this PR is set to be merged in `develop` which is incorrect. We should **branch** `develop` into separate branch. And on release, build from that branch and release and then merge that branch into `master`, while `develop` goes independently.

**Comment by maehne** [comment]
No, I think this is fine. Once this PR gets merged, the version number gets updated and we can fork off the rc-3.6.0 branch from that commit. Then, `develop` is free for new merges of feature PRs. If bug fixes are merged into `develop`, which should be also in the 3.6.0 release, they need to be cherry-picked. Same if bug fixes are made in the `rc-3.6.0` branch, they need to be cherry-picked into `develop`. On `rc-3.6.0` has stabilised, we merge it into `master`.

File: gradle.properties
Comment (maehne) [inline gradle.properties:11]: Shouldn't this be `3.6.0`?

**Review by maehne — CHANGES_REQUESTED** [review]

File: gradle.properties
Comment (BFH-ktt1) [reply gradle.properties:11]: I already branched of with 3.6.0 this morning (branch rc_3_6_0). the 3.7.0 is for the "new" development branch

**Review by maehne — APPROVED** [review]

**Pull request merged**
